cmd/customd/Makefile

```diff
@@ -0,0 +1,30 @@
+.PHONY: all build test image tf protoc clean dist
+
+BUILD_VERSION ?= $(shell git describe --tags)
+BUILD_FLAGS := -mod=readonly -ldflags "-X github.com/iov-one/weave-starter-kit/starterkit.Version=${BUILD_VERSION}"
```

Comment:
`github.com/iov-one/weave-starter-kit/starterkit.Version=${BUILD_VERSION}` did I use this right?

Reply:
I think this should be 
```
-ldflags "-X Version=${BUILD_VERSION}"
```